service/src/main/java/com/codeforcommunity/processor/ProcessorImpl.java

```diff
@@ -13,66 +16,129 @@
 import java.io.Console;
 import java.time.LocalDateTime;
 import java.time.format.DateTimeFormatter;
+import com.codeforcommunity.util.UpdatableBCrypt;
+import java.sql.Timestamp;
 
 public class ProcessorImpl implements IProcessor {
 
   private final DSLContext db;
-  // id should really not be a static number! see down below for dynamic id, hash of username
-  //private int id = 10;
+  // id should really not be a static number! see down below for dynamic id, hash
+  // of username
+  // private int id = 10;
 
   public ProcessorImpl(DSLContext db) {
     this.db = db;
   }
 
   @Override
-  public List<MemberReturn> getAllMembers() {
-    List<Member> members = db.selectFrom(Tables.MEMBER).fetchInto(Member.class);
-    return members.stream().map(member -> new MemberReturn(member.getFirstName(), member.getLastName()))
+  public List<UserReturn> getAllUsers() {
+    List<Users> users = db.selectFrom(Tables.USERS).fetchInto(Users.class);
+    return users.stream()
+        .map(user -> new UserReturn(user.getId(), user.getEmail(), user.getFirstName(), user.getLastName(),
+            user.getGraduationYear().toString(), user.getMajor(), user.getPrivilegeLevel()))
         .collect(Collectors.toList());
   }
 
   @Override
-  public boolean attendMeeting(String meetingid, String username) {
-    Result meetingResult = db.fetch("select * from meeting where id=?;", meetingid);
-    // for now assume username is just the first name
-    Result memberResult = db.fetch("select id from member where first_name=?;", username);
+  public List<UserReturn> getEventUsers(int eventId) {
+    //List<Users> users = db.selectFrom(Tables.USERS).fetchInto(Users.class);
+    List<Users> users = db.fetch("SELECT * FROM users CROSS JOIN (SELECT * FROM event_check_ins where id = ?) AS x;", eventId)
```

Comment:
With jOOQ you should never have to write native SQL queries. Please try to use the fluent jOOQ syntax